src/index.html

```diff
@@ -1,13 +1,350 @@
+              last for generations.
+            </p>
+            <p class="product__price">
+              $ 3199.00
+            </p>
+          </article>
+        </div>
+      </section>
+
+      <section id="section-catalog-preview" class="section-catalog-preview">
+        <h2 class="page__section-title">
+          Sound. Vision.
+          <br>
+          It’s your pick.
+        </h2>
+        <div class="section-catalog-preview__content page__section-content">
+          <article class="product-group" id="speakers">
+            <div class="product-group__img-container">
+              <a href="#">
+                <img
+                  class="product-group__photo"
+                  src="./images/portable-speakers-model-1.png"
+                  alt="portable-speakers-model-1"
+                >
+              </a>
+            </div>
+
+            <div class="product-group__img-container">
+              <a href="#">
+                <img
+                 class="product-group__photo"
+                 src="./images/portable-speaker-model-2.png"
+                 alt="portable-speakers-model-1"
+                >
+              </a>
+            </div>
+
+            <h3 class="product-group__title">Portable speakers</h3>
+          </article>
+
+          <article class="product-group" id="headphones">
+            <div class="product-group__img-container">
+              <a href="#">
+                <img
+                  class="product-group__photo"
+                  src="./images/earphones-model-1.png"
+                  alt="Earphones model-1"
+                >
+              </a>
+            </div>
+
+            <div class="product-group__img-container">
+              <a href="#">
+                <img
+                  class="product-group__photo"
+                  src="./images/earphones-model-2.png"
+                  alt="Earphones model-2"
+                >
+              </a>
+            </div>
+
+            <h3 class="product-group__title">Earphones</h3>
+          </article>
+
+          <article class="product-group">
+            <div class="product-group__img-container">
+              <a href="#">
+                <img
+                  class="product-group__photo"
+                  src="./images/smart-home-model-1.png"
+                  alt="Smart home model-1"
+                >
+              </a>
+            </div>
+
+            <div class="product-group__img-container">
+              <a href="#">
+                <img
+                  class="product-group__photo"
+                  src="./images/smart-home-model-2.png"
+                  alt="Smart home model-2"
+                >
+              </a>
+            </div>
+
+            <h3 class="product-group__title">
+              Smart Home
+            </h3>
+          </article>
+
+          <a class="section-catalog-preview__link-view-all" href="#">
+            <button class="btn">
```

Comment:
The element button must not appear as a descendant of an "a" element accordingly to the living standard.